prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -110,6 +123,116 @@ public async Task<ActionResult<Organization>> CreateOrganization(CreateOrganizat
             );
         }
 
+        // POST: api/Organizations/claim
+        /// <summary>
+        /// Claim an existing Organization.
+        /// </summary>
+        [HttpPost("claim", Name = nameof(ClaimOrganization))]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(ApiResultResponse<int>), StatusCodes.Status200OK)]
+
+        public async Task<ActionResult<Organization>> ClaimOrganization(ClaimOrganizationViewModel claimOrganization)
```

Comment:
Typing ActionResult this way probably does not work the way you might think. What is does is allow you to return _either_ any `ActionResult`, _or_ an object of the specified type. So theoretically with `ActionResult<Organization>` you could do 
```C#
return(organization);
```
rather than 
```C#
return Ok(organization);
```

But this is not actually whats happening: our API responses are wrapped in an `ApiResultResponse` object, so for this to be accurate it would have to be `public async Task<ActionResult<ApiResultResponse<Organization>>> ClaimOrganization`, which is really long and messy. And we aren't even using it, since we call `Ok(whatever)` everywhere.

So I have been pushing to just use `Task<ActionResult>` everywhere since its shorter and we aren't actually using the typed `ActionResult`s